pom.xml

```diff
@@ -3371,5 +3371,25 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>partial-build</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.lesfurets</groupId>
+                        <artifactId>partial-build-plugin</artifactId>
+                        <version>2.6</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <referenceBranch>apache/master</referenceBranch>
+                            <buildAll>true</buildAll>
+                            <skipTestsForNotImpactedModules>true</skipTestsForNotImpactedModules>
```

Comment:
seems great, I would just like to be sure that project are updated when dependencies are changing (for example if you touch jmap, will jmap-integration-tests be played?). It seems to be the case reading the doc, but I'd like to be sure.

Reply:
>  would just like to be sure that project are updated when dependencies are changing

I can confirm it's the case. I manually tested this before the daily.